src/v/kafka/server/group_tx_tracker_stm.cc

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "kafka/server/group_tx_tracker_stm.h"
+
+namespace kafka {
+
+group_tx_tracker_stm::group_tx_tracker_stm(
+  ss::logger& logger, raft::consensus* raft)
+  : raft::persisted_stm<>("group_tx_tracker_stm.snapshot", logger, raft)
+  , group_data_parser<group_tx_tracker_stm>() {}
+
+void group_tx_tracker_stm::maybe_add_tx_begin_offset(
+  kafka::group_id group, model::producer_identity pid, model::offset offset) {
```

Comment:
const ref, or move ?

Reply:
i think you missed that one

Reply:
I adjusted the callers to move the group param, thats what you meant, right?